rpc/dcrd.go

```diff
@@ -228,9 +228,19 @@ func (c *DcrdRPC) GetBestBlockHeader() (*dcrdtypes.GetBlockHeaderVerboseResult,
 		return nil, err
 	}
 
+	blockHeader, err := c.GetBlockHeaderVerbose(bestBlockHash)
+	if err != nil {
+		return nil, err
+	}
+	return blockHeader, nil
+}
+
+// GetBlockHeaderVerbose uses getblockheader RPC with verbose=true to retrieve
+// the header of the requested block.
+func (c *DcrdRPC) GetBlockHeaderVerbose(blockHash string) (*dcrdtypes.GetBlockHeaderVerboseResult, error) {
 	verbose := true
```

Comment:
No need for a local variable here.

Suggest:

```Go
const verbose = true
```

Reply:
Good call, theres a few more of those to update as well.